uploadNeuroDB/bin/pipeline_qc_face.pl

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env perl
+
+############################# MNI Header #####################################
+#@NAME       :  pipeline_qc_face.pl
+#@DESCRIPTION:  Create a QC image showing face of the subject
+#@COPYRIGHT  :
+#              Vladimir S. Fonov  Dec, 2009
+#              Montreal Neurological Institute, McGill University.
+#              Permission to use, copy, modify, and distribute this
+#              software and its documentation for any purpose and without
+#              fee is hereby granted, provided that the above copyright
+#              notice appear in all copies.  The author and McGill University
+#              make no representations about the suitability of this
+#              software for any purpose.  It is provided "as is" without
+#              express or implied warranty.
+###############################################################################
+
+use strict;
+use File::Basename;
+use File::Temp qw/ tempfile tempdir /;
+use File::Path;
+use Getopt::Long;       
+use pipeline_functions;
+
+my $me = basename($0);
+my $verbose = 0;
+my $fake = 0;
+my $clobber = 0;
+my $keep_tmp=0;
+my $stx_xfm;
+
+GetOptions(
+	   'verbose' => \$verbose,
+	   'fake'    => \$fake,
+	   'clobber' => \$clobber,
+     'stx=s'   => \$stx_xfm
+	   );
+
+die "Program usage: ${me} <native> <output> [--stx <xfm>]\n" if $#ARGV < 1 ;
```

Comment:
Since you expect that at this point there should remain exactly two arguments, shouldn't you test for `if $#ARGV == 1` ?

Reply:
indeed!